index.js

```diff
@@ -136,6 +140,36 @@ class Jarvis {
       }
     }
 
+    if (line.startsWith("in this context")) {
+      this.activeConstants = [];
+      const out = 'You are now entering constants. Type the constants, one line at a time. When done, type \'end\'.'
+      return out;
+    }
+
+    if (this.activeConstants) {
+      if (line === 'end') {
+        let keyList = '';
+        this.activeConstants.forEach(constant => {
+          let key = constant.key.toUpperCase();
```

Comment:
the user should define constants as upper case. throw an error if a non uppercase identifier is found.